src/components/sendTo/followAccount.js

```diff
@@ -44,7 +45,19 @@ class FollowAccount extends React.Component {
           disabled={!!this.state.title.error}
           onClick={() => {
             addAccount({ title, address });
-            prevStep();
+            // istanbul ignore else
+            if (typeof this.props.finalCallback === 'function') {
+              this.props.finalCallback();
+              this.props.reset();
+            } else {
+              this.props.nextStep({
+                success: true,
+                title: this.props.t('Success'),
+                body: title + this.props.t(' has been added to your Dashboard'),
```

Comment:
This should be
`body: this.props.t('{{title}} has been added to your Dashboard', { title })`,
because in another language the word order might need to be different.